recipes/pypolyagamma-3/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pypolyagamma-3" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
+
+build:
+  number: 0
+  # We lack gsl on windows
+  skip: True  # [win]
+  skip: True  # [not py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc  # [osx]
```

Comment:
Looks like they are using OpenMP as evidenced by this [line]( https://github.com/slinderman/pypolyagamma/blob/8c4baf70967f454f91cf2ba3e05660b993ab0413/pypolyagamma/pypolyagamma.pyx#L18 ), in which case the selector should be `# [unix]` as it is needed on both platforms.